app/sidekiq/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -69,12 +75,14 @@ class SubmitForm0781 < Job
         if Flipper.enabled?(:form526_send_0781_failure_notification)
           EVSS::DisabilityCompensationForm::Form0781DocumentUploadFailureEmail.perform_async(form526_submission_id)
         end
-
-        ::Rails.logger.warn(
-          'Submit Form 0781 Retries exhausted',
-          { job_id:, error_class:, error_message:, timestamp:, form526_submission_id: }
-        )
       rescue => e
+        cl = caller_locations.first
+        call_location = ZeroSilentFailures::Monitor::CallLocation.new(ZSF_DD_TAG_FUNCTION, cl.path, cl.lineno)
+        form_submission = Form526Submission.find_by(id: form526_submission_id)
```

Comment:
Would you consider wrapping this in its own rescue like I've done here: https://github.com/department-of-veterans-affairs/vets-api/pull/18801/files#diff-dfd5a2e0b3e06d181ef94cdb2737d94a4c67b4efba48fc7f8f5441ecc05a22e8R63-R67

A database call has potential to error (network issue, DB down, record not found, etc) and we really want to reduce the chance of the silent failure incrementing not happening to as low as possible.

Reply:
Addressed in [7fe800b](https://github.com/department-of-veterans-affairs/vets-api/pull/19081/commits/7fe800b564ab7c5e4b3f9e2360a5db09872600d9)